components/Sponza.vue

```diff
@@ -3,8 +3,8 @@
     <p class="sponsor-title">
       Sponsors
     </p>
+    <BecomeSponsorButton />
```

Comment:
I originally meant to make it a reusable component, but a single use component works too

Reply:
oh i see, yeah kinda hard with the birds